benchmark-ph1ex1/convert.py

```diff
@@ -0,0 +1,472 @@
+"""
+This script converts the cross sections in 'OECD-MHTGR350_Simplified.xs'
+to moltres format.
+How to create the XS:
+- Go to main() function
+- Choose one of the 3 options available
+- run 'python convert.py'
+"""
+import numpy as np
+import os
+
+
+def getxs(inFile, index):
```

Comment:
This python file violates most of PEP8, particularly with variable names.

Reply:
trying something else